File: packages/schema/scripts/generate-contract-stamp.ts
import { writeFileSync } from "node:fs";

import {
  contractDigest,
  CONTRACT_STAMP_MODULE,
  CONTRACTS_ROOT,
  renderContractStamp,
} from "../src/contract-digest.ts";

writeFileSync(CONTRACT_STAMP_MODULE, renderContractStamp(contractDigest(CONTRACTS_ROOT)));

File: apps/worker/src/better_answers_worker/generate_contract_stamp.py
from .contract_digest import (
    CONTRACT_STAMP_MODULE,
    CONTRACTS_ROOT,
    contract_digest,
    render_contract_stamp,
)


def main() -> int:
    CONTRACT_STAMP_MODULE.write_text(
        render_contract_stamp(contract_digest(CONTRACTS_ROOT)), encoding="utf-8"
    )
    return 0
